_drafts/concerns-part2.md

```diff
@@ -0,0 +1,179 @@
+Welcome back! [Last time](), we discussed the basics of concerns in Ruby. We then took a detour with the goal of understanding, not just what concerns are, but also how to use them well. As part of that detour, we defined good software as, "software that can change inexpensively," which led us to the following question:
+
+> How do we build software in such a way that change is as inexpensive as possible?
+
+We said that one way to do this is via the use of time-tested software design principles, aka. the topic of today's blog post.
```

Comment:
@tradesmanhelix , looks like part 2 has a theme/topic. So might be good to even have the title of the blog post include the topic of this post rather than a simple "Part 2"